pkg/convert/arb_test.go

```diff
@@ -233,9 +233,7 @@ func Test_ToArb(t *testing.T) {
 			t.Parallel()
 
 			actual, err := ToArb(tt.input)
-			if !assert.NoError(t, err) {
-				return
-			}
+			require.NoError(t, err)
```

Comment:
was that some new linter issue or scope creep ? 🤔

Reply:
new linter https://golangci-lint.run/usage/linters/#testifylint